### PR TITLE
Schedule, Location and Talks&Speakers complete

### PR DESCRIPTION
**Pull request by 571530**

### CODE REVIEW

File: helpers/time.ts [hunk 1]
```diff
@@ -33,8 +38,10 @@ class Time {
 
 function getDuration(type: string) : Time {
     switch(type) {
+        case "Lightning talk": return new Time("00", "10");
         case "Short-talk": return new Time("00", "30");
         case "Long-talk": return new Time("01", "00");
+        case "Workshop": return new Time("01", "30");
```
Comment (Maritt) [inline helpers/time.ts]: workshops har ikke fast tid, kan være 1,5 men også f.eks. 3t

File: pages/components/Room.tsx [hunk 2]
```diff
@@ -1,19 +1,25 @@
-import React from 'react';
-import { colorFromRoomName } from '../../helpers/colors';
-
+import React, { CSSProperties } from 'react';
```
Comment (Maritt) [inline pages/components/Room.tsx]: should both be commaseparated inside the {} ?

File: pages/components/ScheduleEntry.tsx [hunk 3]
```diff
@@ -1,42 +1,45 @@
...
-      .map((talk, i) => talk.speakers
+    let trackIndex = 0;
+    const talks = room.talks
+      .map((talk) => talk.speakers
         .map(speaker => {
           const to = from.copy().add(getDuration(talk.type));
-          const talkEl = (<div className="talk-container" key={i}>
+
+
+          const style = { // For ie support, ie support is far from good.. but this makes i maybe useable
```
Comment (Maritt) [inline pages/components/ScheduleEntry.tsx]: possible to set directly in css?

File: styling/scheduleStyles.scss [hunk 4]
```diff
@@ -184,122 +180,165 @@ div.schedule {
...
+    @media screen and (min-width: $break-schedule) {
+      .talk-container:nth-child(even) .talk {
+        background-color: $dark-bg-color;
+      }
+    }
+    display: flex;
+    flex-direction: column;
+  }
+
   .rooms.multi-track {
```
Comment (Maritt) [inline styling/scheduleStyles.scss]: there is a lot of styling for room, as a room has their own component, is it possible to style it separatly from the schedule?

File: pages/components/ScheduleEntry.tsx (hunk 3 above)
Comment (571530) [reply pages/components/ScheduleEntry.tsx]: Impossible to know which row and column it is in the css so it has to be set here.

File: helpers/time.ts (hunk 1 above)
Comment (571530) [reply helpers/time.ts]: Endret så tid må være eksplisitt satt for workshop

File: pages/components/Room.tsx (hunk 2 above)
Comment (571530) [reply pages/components/Room.tsx]: Change to only no longer import entire namespace and only import parts used

File: styling/scheduleStyles.scss (hunk 4 above)
Comment (571530) [reply styling/scheduleStyles.scss]: Moved room styles into a seperate file

**Pull request merged**